.travis.yml

```diff
@@ -3,14 +3,12 @@ node_js:
 - '0.11'
 before_install:
 - npm install -g npm@latest
+- npm install -g jshint
 - openssl aes-256-cbc -K $encrypted_893d0cdd819b_key -iv $encrypted_893d0cdd819b_iv -in travisapis.enc -out travisapis -d
 script: npm test
-branches:
-only:
-- master
```

Comment:
This will cause the server to rebuild on every commit. I guess it is not related to this though since I will take a look at the deployment pipeline tomorrow